torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1145,8 +1145,8 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
         ((2, 3), (2, 2), (2, 3), False)
     ]
     tests_with_lhs_broadcasting = [
-        ((1,), (2, 2), (2, 3), True),
-        ((), (2, 2), (2, 3), True)
+        ((1,), (2, 2), (2, 2), True),
```

Comment:
Can you keep the old one as well?

Reply:
Sure, that was the original idea, because it doesn't even need broadcasting to exhibit the problem, just square matrices